libs/core/langchain_core/messages/utils.py

```diff
@@ -1424,3 +1425,72 @@ def _convert_to_openai_tool_calls(tool_calls: list[ToolCall]) -> list[dict]:
         }
         for tool_call in tool_calls
     ]
+
+
+DEFAULT_APPROXIMATE_TOKEN_LENGTH = 4
```

Comment:
Why are there global constants for these? Perhaps inline the constant. The key thing is documenting the reason for the constant properly.

Ideally each constant gets a LINK to a reference / short explanation why it's necessary what it does

Reply:
mostly style habit, inline is fine
will add links / explanations